src/site/content/en/blog/loading-ads-page-speed/index.md

```diff
@@ -0,0 +1,207 @@
+authors:
+  - markusbordihn
+hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/27qouRjnUSxESxCIzQp4.jpg
+thumbnail: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/A52ucuDoKR97sMmqxLQQ.jpg
+date: 2023-09-28
+alt: A photograph of a work area with paints and a rubber glove, with a cogwheel painted in red on a wooden surface.
+description: |
+  Learn how to effectively load ads without impacting your page speed, ensuring a seamless user experience, and maximizing revenue opportunities for website owners.
+tags:
+  - blog
+  - performance
+  - web-vitals
+---
+
+Websites heavily rely on online advertising as a primary source of revenue. However, the presence of ads on websites can sometimes come at the expense of user experience and overall page performance. It's therefore vital to strike a balance between monetization and performance for website owners and advertisers, and the user experience.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/4ZPEk7N9pWo3E7iuX1U1.gif", alt="An animated image that illustrates the balance that must be struck when it comes to the amount of ads on a page. As the slider goes from 'no ads' to '100% ads', factors such as visitors, ads, and revenue fluctaute, with revenue and visitors suffering the most when there are 100% ads.", width="660", height="330" %}
+</figure>
+
+Consider a website that extensively places ads within its content, aiming to generate high revenue. However, the overwhelming number of ads frustrates users, leading to a poor user experience and high bounce rates. Despite the potential for substantial revenue from ads, abandonment severely hampers the website's success.
+
+On the other end of the spectrum, consider a website with no ads. This ad-free environment attracts a significant number of users due to its fast loading time and seamless browsing experience. However, without a monetization strategy in place, the website struggles to generate revenue, which may hinder its long-term sustainability and growth.
+
+Both scenarios illustrate the importance of balancing monetization, users and performance.
+
+## Harnessing Core Web Vitals
+
+Passing [the Core Web Vitals thresholds](/defining-core-web-vitals-thresholds/) is essential when it comes to loading ads without negatively impacting page speed. Core Web Vitals, comprising metrics such as [Largest Contentful Paint (LCP)](/lcp/), [First Input Delay (FID)](/fid/) (as well as the upcoming [Interaction to Next Paint (INP) Core Web Vital metric](/inp/) slated to replace FID), and [Cumulative Layout Shift (CLS)](/cls/), are user experience metrics that measure the quality of the user experience of your website.
```

Comment:
Let's just link the main web-vitals page instead of the background page.

Let's also put CLS in the middle so the second sentence isn't as disjointed.

```suggestion
Passing [the Core Web Vitals](/web-vitals/) is essential when it comes to loading ads without negatively impacting page speed. Core Web Vitals, comprising metrics such as [Largest Contentful Paint (LCP)](/lcp/), [Cumulative Layout Shift (CLS)](/cls/), and [First Input Delay (FID)](/fid/) (as well as the upcoming [Interaction to Next Paint (INP) Core Web Vital metric](/inp/) slated to replace FID), are user experience metrics that measure the quality of the user experience of your website.
```